src/main/java/use_case/translate/TranslateInteractor.java

```diff
@@ -9,7 +9,8 @@ public class TranslateInteractor implements TranslateInputBoundary {
     private final TranslateService translateService;
 
     private final TranslateOutputBoundary translatePresenter;
-    public TranslateInteractor(TranslateService translateService, TranslateOutputBoundary translatePresenter) {
+    public TranslateInteractor(TranslateService translateService,
```

Comment:
In my implementation and simialrly to previous coding structures we've seen, Translate isn't creating an entity object with the input or output data, but rather making an output data object for the view. I was wondering how you were thinking of implementing this since I'm thinking the objects for entities should be made beforehand and then sent to the file DAO once that class implements some WordDataAccessInterface. In #24 for me, I created a word object for the interface to store and be accessed later on. 

The coding style and naming conventions are clear! I know we haven't implemented tests yet, but it would be a great addition once your listed TODOs are completed. 

With CA, I'm concerned with the imports since there might be a violation. Use cases shouldn't know about the interface adapters or the API calls. Instead, we should add an abstraction layer so the use cases trusts that whatever api interface adapter it'll call will return what it needs, without knowing exactly what type it is, etc. What I've done for this is created classes with my APIs that will implement the interfaces from entities since the dependencies should point inwards.